AppSearchSample/app/src/main/java/com/android/example/appsearchsample/model/Note.kt

```diff
@@ -41,5 +42,10 @@ data class Note(
   @Document.StringProperty(
     indexingType = AppSearchSchema.StringPropertyConfig.INDEXING_TYPE_PREFIXES
   )
-  val text: String
+  val text: String,
+
+  @Document.StringProperty(
+    indexingType = AppSearchSchema.StringPropertyConfig.INDEXING_TYPE_PREFIXES
+  )
+  val color: String
```

Comment:
I believe bolding exact matches is a helpful AppSearch feature to demonstrate in our sample, but I'm less convinced adding a color component to the UI demonstrates anything critical to understanding AppSearch. What is the motivation for adding this to the main branch?